tika-parsers/src/main/java/org/apache/tika/parser/ner/mitie/MITIENERecogniser.java

```diff
@@ -0,0 +1,160 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.tika.parser.ner.mitie;
+
+
+import org.apache.tika.parser.ner.NERecogniser;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.lang.reflect.Method;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.Map;
+import java.util.HashMap;
+import java.util.ArrayList;
+
+/**
+ *  This class offers an implementation of {@link NERecogniser} based on
+ *  trained models using state-of-the-art information extraction tools. This NER requires additional setup,
+ *  due to runtime binding to MIT Information Extraction.
+ *  See <a href="http://wiki.apache.org/tika/TikaAndMITIE">
+ *      Tika MITIE Wiki</a> for configuring this recogniser.
+ *  @see NERecogniser
+ *
+ */
+public class MITIENERecogniser implements NERecogniser {
+
+    private static final Logger LOG = LoggerFactory.getLogger(MITIENERecogniser.class);
+
+    public static final String MODEL_PROP_NAME = "ner.mitie.model";
+
+    public static final Set<String> ENTITY_TYPES = new HashSet<String>(){{
+        add(PERSON);
+        add(LOCATION);
+        add(ORGANIZATION);
+        add("MISC");
+    }};
+
+    private static final String NamedEntityExtractor_Class = "edu.mit.ll.mitie.NamedEntityExtractor";
+    private boolean available = false;
+    private Object extractorInstance;
+
+    public MITIENERecogniser(){
+        this(System.getProperty(MODEL_PROP_NAME));
+    }
+
+    /**
+     * Creates a NERecogniser by loading model from given path
+     * @param modelPath path to NER model file
+     */
+    public MITIENERecogniser(String modelPath) {
+        try {
+            if(!(new File(modelPath)).exists()) {
+                LOG.warn("{} does not exist", modelPath);
+            }else {
+                Class<?> namedEntityExtractorClass = Class.forName(NamedEntityExtractor_Class);
+                extractorInstance = namedEntityExtractorClass.getDeclaredConstructor(new Class[]{String.class}).newInstance("/Users/manali/cs599_dr/MITIE/MITIE-models/english/ner_model.dat");
```

Comment:
@manalishah
 Please use path from function parameter instead of the hardcoded string.
 Perhaps you forgot to modify in last moment  ;-) ?